libminifi/src/utils/StringUtils.cpp

```diff
@@ -19,6 +19,8 @@
 
 #include "utils/Environment.h"
 
+#include <limits>
```

Comment:
`<limits>` should be already included in this chain: `StringUtils.h -> gsl.h -> gsl-lite.hpp`.

Reply:
@hunyadi-dev this is seen when compiling with upcoming GCC 11